clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3014,6 +3014,13 @@ static void handleSubGroupSize(Sema &S, Decl *D, const ParsedAttr &AL) {
   if (D->getAttr<IntelReqdSubGroupSizeAttr>())
     S.Diag(AL.getLoc(), diag::warn_duplicate_attribute) << AL;
 
+  if (AL.getAttributeSpellingListIndex() ==
+      IntelReqdSubGroupSizeAttr::CXX11_cl_intel_reqd_sub_group_size) {
```

Comment:
Last question, I promise:)  DO we also intend to deprecate the GCC spelling (as intel_reqd_sub_group_size) ?  It seems silly (though not my call) to deprecate 1 but not the other.

Reply:
It is alight Erich. 

>>DO we also intend to deprecate the GCC spelling (as intel_reqd_sub_group_size) ?

My understanding is here that no, we do not intend to deprecate the GCC spelling (as intel_reqd_sub_group_size).

Reply:
This comment is showing unresolved. @erichkeane Do you have still have question on this?

Reply:
No, I don't.